app/src/main/java/com/example/wnews/services/AuthService.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.wnews.services
+
+import com.example.wnews.models.Comment
+import com.example.wnews.models.UserAuth
+import retrofit2.Call
+import retrofit2.http.*
+
+
+interface AuthService {
+
+    @POST("/auth/sign_in")
+    fun createUser(@Header("email") email:String,@Header("password") password:String): Call<UserAuth?>?
```

Comment:
Make sure to be consistent with spaces between variables. This is just some styling thing, but please notice that some variables in this file got spaces between them and some others don't.

Ideally, this should be automated using an automated tool called a `linter`, which checks for styling issues. Usually on web development we've got tools like Prettier or Prettify that take care of this thing. 

On a quick Google search, I found this Prettier plugin for Android Studio that may help you automatically fix styling issues in your files. Check it out or look for any other that you may like more. 
https://plugins.jetbrains.com/plugin/7405-android-studio-prettify